images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -44,10 +44,8 @@ RUN if [ "${DEBIAN_FLAVOR}" = "bullseye" ]; then \
             libcurl4 \
             libssl1.1 \
             libyaml-dev \
-        && rm -rf /var/lib/apt/lists/* \
-        && curl -LO http://security.debian.org/debian-security/pool/updates/main/libx/libxml2/libxml2_2.9.10+dfsg-6.7+deb11u3_amd64.deb \
```

Comment:
not sure why we are pulling from this manual source to install. I've updated to `libxml2-dev` which we use elsewhere in our codebase. The benefit of this is that we no longer need to manually bump these 3 lines when they release a new libxml2 version

Reply:
I looked into the original PR that introduced this (https://github.com/microsoft/Oryx/pull/1483) and couldn't find the reason. Maybe something related to Bullseye? Might be worth double checking with @cormacpayne in case he remembers this PR and asked Stella about that maybe?
I can see that libxml2 as installed before targeted 2.9.10, when the one you changed it to is 2.9.4 I believe. Maybe that's the reason? Also, do you know why we use libxml2-dev and not libxml2?

Reply:
I also looked at that PR and didn't see any indications, but potentially a PHP dep? Yeah @cormacpayne if you remember anything once you are back, let us know!

`when the one you changed it to is 2.9.4 I believe`
Curious where you are seeing this? I thought that would install 2.9.10: https://packages.debian.org/stable/libxml2-dev

As for `libxml2-dev` vs `libxml2`, I installed it as it is generally used a few different places in our code. That said, its 2MB larger than `libxml2` so I'm going to try to swap them and see if we can get through our nightly testing suite.

Reply:
Thanks, sounds good. Regarding 2.9.10 vs. 2.9.4: you're right, 2.9.10 will indeed be installed with this.

Reply:
@pauld-msft @daniv-msft -- unfortunately I can't find any additional context on this, but it looks like it was a temporary holdover while we couldn't publish bullseye binaries, so I'm assuming it was a library that was bundled with the buster binaries that was available in some layer that wasn't available in the bullseye flavor of the GitHub Actions build image. We should be good now since all of this has been resolved with bullseye binaries 👍

Reply:
Thanks @cormacpayne!